File: Algorithm/Programmers/Python/Level 1/최소직사각형.py
def solution(sizes):
    q = [0, 0]
    for i in sizes:
        i.sort()
    for i in sizes:
        for j in range(2):
            if q[j] < i[j]:
                q[j] = i[j]
    return q[0] * q[1]

File: Algorithm/Programmers/Java/Level 1/최소직사각형.java
class 최소직사각형 {
    public int solution(int[][] sizes) {
        int[] comparer = { 0, 0 };
        for (int[] arr : sizes) {
            int a = Math.min(arr[0], arr[1]);
            int b = Math.max(arr[0], arr[1]);
            comparer[0] = Math.max(a, comparer[0]);
            comparer[1] = Math.max(b, comparer[1]);
        }
        return comparer[0] * comparer[1];
    }
}